src/components/AttentionBox/AttentionBox.jsx

```diff
@@ -82,22 +90,27 @@ const AttentionBox = ({
             ariaHidden
             clickable={false}
             icon={icon}
-            className={cx(`${baseClassName}__title-container__icon`, `${classNameWithType}__title-container__icon`)}
+            className={cx(
+              `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__icon`,
+              `${classNameWithType}__title-container__icon`
+            )}
             ignoreFocusStyle
             iconLabel={iconLabel}
           />
         )}
         {text}
       </div>
       {onClose && (
-        <Icon
-          iconType={Icon.type.SVG}
-          iconLabel="Close"
+        <IconButton
+          size={IconButton.sizes.SMALL}
+          color={IconButton.colors.ON_PRIMARY_COLOR}
+          className={cx(`${ATTENTION_BOX_CSS_BASE_CLASS}__close-icon`, "icon")}
```

Comment:
why icon class name?

Reply:
it's super generic class name

Reply:
`wrapper` is also super generic class name)
But is there a difference if it's nested under `close-icon` rule? It can't be used without it, no?

Reply:
I did not see a wrapper class name but it is also bad if it's exist.
Let's not use super generic class names before converting to css modules because there is always a change it will cause a weird behavior (even if it's nested maybe other deveper use the same class name and you will get some of his defentions)

Reply:
Fixed